student.rb

```diff
@@ -9,6 +9,11 @@ def initialize(age, classroom, name = 'Unknown', parent_permission: true)
     @classroom = classroom
   end
 
+  def add_classroom(classroom)
+    @classroom = classroom
+    classroom.students.push(self) unless classroom.students.include?(self)
```

Comment:
- [ ] This will throw an error because in `Classroom` class an array of students is  `student` not `students`